tests/virt/cluster/strict_reconcile/test_strict_reconcile.py

```
@@ -63,9 +63,10 @@ def restore_and_log_error(
     actual_value,
 ):
     updated_resource.restore()  # Only restore explicitly, if virt-operator fails to revert automatically.
-    LOGGER.error(f"Timeout waiting for {resource.kind}: {resource.name} resource being reconciled.")
-    LOGGER.error(f"Expected: {expected_value}\n Actual: {actual_value}")
-    raise
+    raise TimeoutExpiredError(
+        f"Timeout waiting for {resource.kind}: {resource.name} resource being reconciled."
+        f" Expected: {expected_value}, Actual: {actual_value}"
+    )
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)